external/llvm-project/mlir/include/mlir/Dialect/LLVMIR/ROCDLOps.td

```diff
@@ -131,6 +131,18 @@ def ROCDL_BarrierOp : ROCDL_Op<"barrier"> {
   let assemblyFormat = "attr-dict";
 }
 
+def ROCDL_SetPrioOp : ROCDL_IntrOp<"s.setprio", [], [], [], 0>,
+  Arguments<(ins I16:$priority)> {
+  let results = (outs);
+  let assemblyFormat = "$priority attr-dict `:` type($priority)";
```

Comment:
You don't actually have to give the type here - since it's always I16/I32, it'll be inferred